emoji/build.gradle

```diff
@@ -0,0 +1,45 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+
+    defaultConfig {
+        minSdkVersion 21
```

Comment:
With 3 modules, we should add `minSdk` to `dependencies.gradle`

Reply:
Agree. Will do. About the versioning, in the end, are we following any versioning pattern for the internal modules? @luciofm @filipedelimabrito

Reply:
Since they are internal, we don't really care about versioning. If you wan't we could move the versioning to `dependencies.gradle` and use the same for all modules, but I don't see any advantage, or need for this,